src/dataset-builder/ConceptSearch.ts

```diff
@@ -33,15 +34,24 @@ const DebouncedTextInput = withDebouncedChange(TextInput);
 export const ConceptSearch = (props: ConceptSearchProps) => {
   const { initialSearch, domainOption, onCancel, onCommit, onOpenHierarchy, actionText, datasetId, initialCart } =
     props;
-  const [search, setSearch] = useState<string>(initialSearch);
+  const [searchText, setSearchText] = useState<string>(initialSearch);
```

Comment:
Why change the state name?

Reply:
Personally I prefer `search` unless there's a need to disambiguate between different kinds of `search` objects inside this scope. But I think I prefer shorter names overall, maybe more then others do.